cmake/compiler/arcmwdt/compiler_flags.cmake

```diff
@@ -210,3 +210,9 @@ set_compiler_property(PROPERTY warning_shadow_variables)
 
 set_compiler_property(PROPERTY no_builtin -fno-builtin)
 set_compiler_property(PROPERTY no_builtin_malloc -fno-builtin-malloc)
+
+# Compiler flag for placing functions in their own sections:
```

Comment:
This comment is wrong, cause the `no_function_sections` prevents placing functions in their own sections.